### ISSUE TITLE
add validation set option

### ISSUE THREAD

**Issue by snowformatics** [snowformatics/gwastic_desktop#14]
add validation set option for ML methods

**Comment by snowformatics** [comment]
add also information which id was in training set
